src/tests/suite/daos_rebuild.c

```diff
@@ -1748,9 +1748,9 @@ rebuild_fail_all_replicas(void **state)
 static void
 multi_pools_rebuild_concurrently(void **state)
 {
-#define POOL_NUM		6
-#define CONT_PER_POOL		4
-#define OBJ_PER_CONT		256
+#define POOL_NUM		4	
+#define CONT_PER_POOL		2	
+#define OBJ_PER_CONT		8	
```

Comment:
(style)  trailing whitespace

Reply:
Please remove the trailing whitespaces, also I'm OK to reduce the number of pools and containers, but do we want ot reduce object per container to 8? This is a too small number to me.

Reply:
oh, I want to run it with the vm and also shrink the time. since this test only verified the rebuild for multiple pools/containers. so I put the number low for now. When we only run rebuild over real hardware, I will increase the number again.